lib/shoulda/matchers/action_controller/permit_matcher.rb

```diff
@@ -62,9 +62,10 @@ module ActionController
       #             password: 'password'
       #           }
       #         }
-      #         should permit(:first_name, :last_name, :email, :password).
+      #         assert_accepts permit(:first_name, :last_name, :email, :password).
       #           for(:create, params: params).
-      #           on(:user)
+      #           on(:user),
+      #           subject
```

Comment:
Sure, should I still leave out the parens? I'm not sure what the right style is here.